developer/src/kmc-ldml/build.sh

```diff
@@ -75,14 +75,19 @@ function do_build_fixtures() {
 
   # Generate a binary file from basic.txt for comparison purposes
   node ../../../common/tools/hextobin/build/hextobin.js ./test/fixtures/basic.txt ./build/test/fixtures/basic-txt.kmx
+  if ! cmp ./build/test/fixtures/basic-xml.kmx ./build/test/fixtures/basic-txt.kmx; then
+    printf "${COLOR_RED}basic-xml.kmx and basic-txt.kmx are different!${COLOR_RESET}\n"
```

Comment:
This should be a `builder_die` to fail here. It's a problem to fix

```suggestion
    builder_die "basic-xml.kmx and basic-txt.kmx are different!"
```

Reply:
i went with this, and also added some hints if someone randomly gets this message in the future (mentioning basic.txt and basic.xml)